README.md

```diff
@@ -1,79 +1,94 @@
+
+
+
 # Unfurl
 
-This project contains reusable preview  for Unfurl
+  
+Lightning fast, resuable and framework agnostic component, for providing link previews in several modes and layouts.
 
 ## Getting Unfurl Free API Token
 
+  
+
 * Signup on <https://unfurl.io/> or Signin if you are already registered.
 
+  
+
 * After login, from the dashboard navigate to Generate/Manage your Api Tokens here.
 
-* Next, in Create API Token section assign a token name and click on create.
+  
 
-> Note: Please save the generated api token, it is only displayed for one time.  
+* Next, in Create API Token section assign a token name and click on create.
 
-## Seting up API token
+  
 
-* Register your API token in `apps\demo\src\app\app.module.ts` file
+> Note: Please save the generated api token, it is only displayed for one time.
 
-    ```typescript
-        @NgModule({
-            declarations: [AppComponent],
-            imports: [
-                BrowserModule,
-                HttpClientModule,
-                PreviewModule.forRoot({
-                /* Free Test Token */
-                apiToken: 'Paste your API Token Here'
-                })
-            ],
-            providers: [],
-            bootstrap: [AppComponent]
-        })
-    ```
+  
 
 ## Preview Your URL & setup design modes for preview
 
-### Test Your First URL preview
+ 
+
+ ### Installation 
+ -  **Using NPM**
+	 - `npm install @unfurl/elements`
+ - **Using Yarn**
+	 -  `yarn install @unfurl/elements`
+
+### Getting Started
+Once installation is done, it can be imported and configured as follows.
 
-* Navigate to `apps\demo\src\app\app.component.html` and paste your URL in `<unfurl-preview url="Your URL">` tag.
+- `import '@unfurl/elements'`
+- **Typescript**
+```typescript
+window.Unfurl.setConfig({  
+    apiToken: 'API_TOKEN'  
+  });
+```
+- **For other environments**
+```javascript
+Unfurl.setConfig({  
+    apiToken: 'API_TOKEN'  
+  });
 
-    ```html
-        <div class="spacer"></div>
-        <unfurl-preview url="Your URL" mode="simple"></unfurl-preview>
-        <div class="spacer"></div>
-        <unfurl-preview url="Your URL" mode="compact"></unfurl-preview>
-        <div class="spacer"></div>
-        <unfurl-preview url="Your URL" mode="detailed"></unfurl-preview>
-        <div class="spacer"></div>
-   ```
+```
+
+* Paste your URL in `<unfurl-preview url="Your URL">` tag.
```

Comment:
Write here,
use <unfurl-preview></unfurl-preview> inside your template, where you want the link to be previewed. 
Paste your URL in the URL attribute.

Reply:
Done.